core/api/src/main/java/org/trellisldp/api/NoopAccessControlService.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.trellisldp.api;
+
+import static java.util.Arrays.asList;
+import static java.util.stream.Collectors.toSet;
+import static org.trellisldp.api.TrellisUtils.getInstance;
+
+import java.util.Set;
+
+import org.apache.commons.rdf.api.IRI;
+import org.apache.commons.rdf.api.RDF;
+
+/**
+ * A no-op AccessControlService implementation.
+ */
+public class NoopAccessControlService implements AccessControlService {
+
+    private static final RDF rdf = getInstance();
+    private static final String URI = "http://www.w3.org/ns/auth/acl#";
+    private static final Set<IRI> modes = asList("Control", "Read", "Write", "Append").stream()
```

Comment:
I know it's Guava, but seems like ` ImmutableSet.of(rdf.createIRI("Control"), ...)` is a lot clearer here and it is also guaranteed immutable.

Reply:
I don't really want to have the API depend on Guava. What if we use something like `Collections.unmodifiableSet(...)`. The creation of it isn't as clean as with Guava, but it doesn't add the extra dependency.

Reply:
Oh, I didn't realize it wasn't. For some reason I thought it came along with some Commons library. No, in that case `Collections.unmodifiableSet` would be great.